### PR TITLE
Indicate that disposed tokens are not cancelable

### PR DESCRIPTION
**Pull request by AArnott**
As discussed in #22

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (conradreuter) [inline babel.config.js:1]: Is Babel needed for `get`, or why is this added?

File: src/CancellationToken.spec.ts [hunk 2]
```diff
@@ -143,6 +143,19 @@ describe('A cancellation token', () => {
...
+      dispose()
+      expect(token.canBeCancelled).toBeFalsy()
+    })
+
+    it('should report being canceled and cancelable after its source is disposed', () => {
+      cancel()
+      dispose()
+      expect(token.canBeCancelled).toBeTruthy()
+      expect(token.isCancelled).toBeTruthy()
+    })
```
Comment (conradreuter) [inline src/CancellationToken.spec.ts]:
```suggestion
    it('should report being uncancellable after its source is disposed', () => {
      expect(token.canBeCancelled).toBeTruthy()
      dispose()
      expect(token.canBeCancelled).toBeFalsy()
    })

    it('should report being cancelled and cancellable after its source is disposed', () => {
      cancel()
      dispose()
      expect(token.canBeCancelled).toBeTruthy()
      expect(token.isCancelled).toBeTruthy()
    })
```

**Comment by conradreuter** [comment]
@AArnott I added a minor spelling suggestion in a commit. Would you mind just squashing that into your commit? 🙂

**Comment by AArnott** [comment]
@conradreuter I've squashed your commit into mine.
